content/posts/2024-02-26-GSoC.md

```diff
@@ -0,0 +1,74 @@
+---
+author: "Anton Korobeynikov"
+date: "2024-02-26"
+tags: ["GSoC"]
+title: "LLVM Google Summer of Code 2024 & 2023"
+---
+
+The LLVM organization was accepted to participate in Google Summer of Code in 2024. Soon, prospective participants will begin submitting their project proposals, and mentors will review them to select those who will spend a significant amount of time this year contributing to various parts of LLVM.
+
+But for now, let's look back and see what we had in 2023. The Google Summer of Code 2023 was very successful for the LLVM project. Overall, we received 54 proposals for 24 open projects. Out of this set of proposals, 20 projects went to a successful finish, covering different aspects of LLVM and its subprojects.
```

Comment:
I would change to "But first, let's look..." 
Change "went to a successful finish, covering" to "were successfully completed and covered many different aspects of.."